tests/unit/alexnet_model.py

```diff
@@ -101,11 +101,11 @@ def cifar_trainset(fp16=False):
     dist.barrier()
     if local_rank != 0:
         dist.barrier()
-
     data_root = os.getenv("TEST_DATA_DIR", "/tmp/")
-    trainset = torchvision.datasets.CIFAR10(root=os.path.join(data_root, "cifar10-data"),
+    download = False if os.getenv("CIFAR10_DATASET_PATH") else True
```

Comment:
I recommend simplification like the following to improve readability. I have approved regardless. 
```suggestion
    if os.getenv("CIFAR10_DATASET_PATH"):
       data_root = os.getenv("CIFAR10_DATASET_PATH")
       download = False
    else:
       data_root = os.path.join(os.getenv("TEST_DATA_DIR", "/tmp"), "cifar10-data")
       download = True
```

Reply:
@raza-sikander - does this resolve your issues, could you confirm before we commit these changes or just merge the branch as is?

Reply:
Updating the commit as per suggestion for simplification